session/session_test.go

```diff
@@ -3138,6 +3138,23 @@ func (s *testSessionSuite2) TestStmtHints(c *C) {
 	tk.MustExec("select /*+ READ_CONSISTENT_REPLICA(), READ_CONSISTENT_REPLICA() */ 1;")
 	c.Assert(tk.Se.GetSessionVars().StmtCtx.GetWarnings(), HasLen, 1)
 	c.Assert(tk.Se.GetSessionVars().GetReplicaRead(), Equals, kv.ReplicaReadFollower)
+
+	// Test SET_VAR hint
```

Comment:
could you add more tests to cover all the updatable hints? Maybe we can write a separate test suite for updatable and non-updatable hints to speed up unit test?